packages/fleet_server/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.1"
+  changes:
+    - description: Fix agent template to avoid server:null
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/5896
```

nit. updating PR link
```suggestion
      link: https://github.com/elastic/integrations/pull/6555
```